render_pdf/src/runscript

```diff
@@ -0,0 +1,5 @@
+python make_latex.py
```

Comment:
better name -- maybe `create_pdf.sh`, give it the `.sh` extension and add `#!/usr/bin/sh` at the top of the file, then `chmod +x create_pdf.sh` to mark it as executable